sidekiq-undertaker.gemspec

```diff
@@ -42,13 +44,13 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rspec-core", "~> 3.8"
   spec.add_development_dependency "rspec-mocks", "~> 3.8"
   spec.add_development_dependency "rspec-sidekiq", "~> 3.0"
-  spec.add_development_dependency "rt_rubocop_defaults", "~> 1.2"
-  spec.add_development_dependency "rubocop", "~> 0.77"
-  spec.add_development_dependency "rubocop-rspec", "~> 1.30"
+  spec.add_development_dependency "rt_rubocop_defaults", "~> 2.1"
```

Comment:
Wait until https://github.com/runtastic/rt_rubocop_defaults/pull/13 is merged and released